codelists/forms.py

```diff
@@ -108,7 +108,7 @@ def clean_csv_data(self):
             )
 
         cleaned_header = [
-            '"' + h.strip('" ') + '"' if h[0] == '"' and h[-1] == '"' else h.strip()
+            '"' + h[1:-1].strip() + '"' if h[0] == '"' and h[-1] == '"' else h.strip()
```

Comment:
This explicitly only removes the first and last chars of the string, which are the ones being tested for presence of quotes. Notwithstanding a library to do this for us, I think this might be right? @madwort